src/modules/tests/commentCounter.test.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable*/
+
+ import { counterComment } from "./mocks/commentCounter";
+
+ test('comment should be = 6', () => {
+   const div = document.createElement('p');
+   div.innerHTML = `
+   <p class="comments"></p>
+   <p class="comments"></p>
+   <p class="comments"></p>
+   <p class="comments"></p>
+   <p class="comments"></p>
+   <p class="comments"></p>
+   `;
+ 
+   document.body.appendChild(div);
+ 
+   const n = counterComment();
+ 
+   expect(n).toEqual(6);
+ });
```

Comment:
- [ ] Kindly make sure all the tests are running successfully, when I checked this test it failed, I got the following message:

![image](https://user-images.githubusercontent.com/20567503/188245208-ef98033d-883e-4f61-95a2-7cc7c5f75bde.png)

Kindly fix this then submit again!

Reply:
Oh, you were right! With the curly brackets, the test couldn't call the default function, It happened to me earlier so I was able to fix it fast!